packages/adblocker-webextension/adblocker.ts

```diff
@@ -396,16 +397,17 @@ export class WebExtensionBlocker extends FiltersEngine {
 
       // Inject scripts from content script
       if (scripts.length !== 0) {
-        sendResponse({
+        response = {
```

Comment:
Is the promise API supported by Chrome without the polyfill?

Reply:
I have tested it on Chrome, and the fix works there.

Reply:
Was trying out the change, but still got the error in the Ghostery Extension. That means, either my test setup is faulty (which could well be!), or there are other places, or it is not working for other reasons.

But now I saw that it is being discussed on the webextension-polyfill library. I left a comment here: https://github.com/ghostery/ghostery-extension/issues/806#issuecomment-1195282088

Long story short, I think we should wait for their fix to get merged and released. 

With the change in our PR here, there is a small change in semantic which might have unintended side-effects: instead of immediately sending the script, we are first waiting for the style injections to resolve. Since scriptlet injection is a time-sensitive topic, I'm a bit afraid to change it.